relevanceai/resources/agent.py

```diff
@@ -151,7 +151,7 @@ def rerun_task(
         self,
         conversation_id: str,
     ) -> Optional[TriggeredTask]:
-        trigger_message_data = self._get_trigger_message(agent_id, conversation_id)
+        trigger_message_data = self._get_trigger_message(self.agent_id, conversation_id)
```

Comment:
_get_trigger_message only takes conversation_id? Do we still need agent_id here

Reply:
yep you're right. just fixed!